multimodal/src/autogluon/multimodal/utils/model.py

```diff
@@ -556,3 +557,15 @@ def modify_duplicate_model_names(
 
 def list_timm_models(pretrained=True):
     return timm.list_models(pretrained=pretrained)
+
+
+def is_lazy_weight_tensor(p: Tensor) -> bool:
+    from torch.nn.parameter import UninitializedParameter
+
+    if isinstance(p, UninitializedParameter):
+        warnings.warn(
```

Comment:
Wondering how frequent this gonna happen? If it is the case, shall we just run one dummy forward pass to make these layers initialized and then calculate?

Reply:
This refers to pytorch lightning: https://github.com/Lightning-AI/lightning/blob/984f49f7195ddc67e961c7c498ee6e19fc0cecb5/src/lightning/pytorch/utilities/model_summary/model_summary.py#L236
In some cases, parameters are uninitialized.

Reply:
ok, as long as we are consistent with lightning, shall be good